se3cnn/SO3.py

```diff
@@ -563,7 +571,7 @@ def clebsch_gordan(l1, l2, l3):
         return _clebsch_gordan(l3, l1, l2).transpose(0, 2).transpose(0, 1).contiguous()
 
 
-@cached_dirpklgz("cache/clebsch_gordan")
+@cached_dirpklgz(user_cache_dir("se3cnn/clebsch_gordan"))
 def _clebsch_gordan(l1, l2, l3, _version=2):
```

Comment:
change to `_version=3` if the output of the function my change

Reply:
incremented